rasa/shared/core/trackers.py

```diff
@@ -589,10 +590,31 @@ def update(self, event: Event, domain: Optional[Domain] = None) -> None:
             for e in domain.slots_for_entities(event.parse_data["entities"]):
                 self.update(e)
 
+    def update_with_events(
+        self,
+        new_events: List[Event],
+        domain: Optional[Domain],
+        override_timestamp: bool = True,
+    ) -> None:
+        """Adds multiple events to the tracker.
+
+        Args:
+            new_events: Events to apply.
+            domain: The current model's domain.
+            override_timestamp: If `True` reset all timestamps of the events. As the
```

Comment:
Maybe `actualize` instead of `reset`? Reset sounds to me like "set to zero".

Reply:
Changed it to `refresh`.